pages/tutorials/[slug].js

```diff
@@ -1,49 +1,69 @@
 import fs from 'fs'
 import path from 'path'
 import matter from 'gray-matter'
-import { marked } from 'marked'
 import styled from 'styled-components'
 import DynNavbar from '../../components/DynNavbar'
 import Footer from '../../components/Footer'
+import { serialize } from 'next-mdx-remote/serialize'
+import { MDXRemote } from 'next-mdx-remote'
+import remarkMath from 'remark-math'
+import rehypeKatex from 'rehype-katex'
+import rehypeHighlight from 'rehype-highlight'
+import Head from 'next/head'
+
 
 
 const Hero = styled.div`
   display: flex;
-  justify-content: center;
+  justify-content: left;
   background: #fff;
-  align-items: center;
+  align-items: left;
   padding-left: 10%;
   padding-right: 10%;
   padding-top: 20px;
   padding-bottom: 20px;
 `
 
+
+
+
 export default function PostPage({
-  frontmatter: { title, cover_image },
-  slug,
-  content,
+  mdxSource,
 }) {
+
+
   return (
     <>
+    <Head>
+      <link
+        rel="stylesheet"
+        href="https://cdn.jsdelivr.net/npm/katex@0.13.11/dist/katex.min.css"
```

Comment:
If possible, use the CSS in the `katex` package, rather than the `<link rel="stylesheet">`.
It is better because in this way it gets compiled (bundled) with the rest of the page, with two advantages:
1. it is served directly with the page, you don't need to wait for an extra load step
2. you do not depend on https://cdn.jsdelivr.net being available (while is a popular CDN and rather fast, we already depend on GitHub Pages, and one dependency is better than two)

If not possible, I believe you're not using `katex` package at all (at least directly) so we should drop it from `package.json`.